x-hooks/use-api.tsx

```diff
@@ -270,6 +270,12 @@ export default function useApi() {
       })
   }
 
+  async function createReviewForPR(issueId: string, pullRequestId: string,  githubLogin: string, body:string) {
+    return client.put('/api/pull-request/review', {issueId, pullRequestId, githubLogin, body})
+      .then(response => response)
+      .catch(error => { throw error })
```

Comment:
This already happens (as in: this catch function is the default one), you don't need to - you can just catch on the other side when you use the function